app-creator/client/src/utils/template-generation.ts

```diff
@@ -0,0 +1,116 @@
+import { AppCreatorStore, WidgetMetaData } from '../redux/reducer';
```

Comment:
This code was moved from story-board.ts to here so it can be shared across the applications. It contains the logic for rendering a template given its corresponding string.